docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_w_image.scss

```diff
@@ -1,3 +1,11 @@
+// An note on the functionality of images
```

Comment:
ah sorry. should this content go on line 87 or 112?

Reply:
I like it on line 87. 😄